### ISSUE TITLE
UI issue within forms containing checkboxes

### ISSUE THREAD

**Issue by kruthis123** [okta/okta-signin-widget#3257]
### Describe the bug

Within forms, whenever the text associated with a checkbox is longer than 4 or 5 lines, duplicate checkbox appears. We could spot this issue on pages that extend FormController. On looking further we found out custom checkbox is displayed as a background image which is a css sprite image.

<img width="1792" alt="Form checkbox ui issue" src="https://github.com/okta/okta-signin-widget/assets/52000218/0cc47bb7-3bbc-43b9-bd0f-80d723dc3ff5">

### Reproduction Steps

This bug should be reproducible on any form that displays checkboxes and the label for the input (for checkbox) is very long. We could spot the issue on the granular-consent page which can be reproduced as follows -

1. Create an optional scope with a long description and include it in the access policy for an app.
2. Form an authorize request for the app including the above scope and try signing in with a user.
3. The granular-consent screen will come up asking for permission from user for the scope.
4. Edit the html and add a p html tag below the label for a particular scope. Change the innerHtml of the p tag to contain the long description.

### SDK Versions

  System:
    OS: macOS 13.3
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 829.41 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.19.2 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 8.15.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 113.0.5672.126
    Firefox: 113.0.2
    Safari: 16.4
  npmPackages:
    @okta/eslint-plugin-okta: 0.5.0 => 0.5.0 
    @okta/okta-auth-js: ^6.0.0 => 6.0.0

**Comment by jaredperreault-okta** [comment]
@kruthis123 what widget version are you using? Doesn't seem to be returned in the `npx` command output

**Comment by kruthis123** [comment]
Widget Version -  6.2.0 and 7

```
<div data-se="o-form-fieldset-optedScopes.offline_access" class="o-form-fieldset o-form-label-top">
    <div data-se="o-form-input-container" class="o-form-input">
        <span data-se="o-form-input-optedScopes.offline_access" class="o-form-input-name- 
          optedScopes.offline_access">
            <div class="custom-checkbox">
                <input type="checkbox" name="optedScopes.offline_access" id="input64">
                <label for="input64" data-se-for-name="optedScopes.offline_access" class="checked fe_f_all" style="font- 
                  family: &quot;Source Sans Pro&quot;, arial, sans-serif; font-size: 18px;">
                      <b class="no-translate">Have offline access</b>
                      <p class="no-translate">Allows this application to access your data when you aren't using the application. 
                      </p>
                 </label>
            </div>
        </span>
    </div>
</div>
```

We suspect the issue to be happening due to the following style within /assets/sass/modules/_forms.scss where a css sprite is used as a background image without restricting the height.
```
.custom-checkbox {
  label {
    background-image: url('../img/ui/forms/checkbox-sign-in-widget.png');
  }

  label.focus {
    background-position: -16px -1013px;
  }

  label.focus::before {
    background-position: -16px -1020px;
  }
}
```

**Comment by denysoblohin-okta** [comment]
Thanks for reporting this issue.
Internal ref: OKTA-614475

As a workaround, you could use CSS fix like this
```css
#okta-sign-in .siw-main-view .custom-checkbox label {
  clip-path: polygon(0 0, 0 30px, 20px 30px, 20px 100%, 100% 100%, 100% 0);
}
```

**Comment by glenfannin-okta** [comment]
@kruthis123 #3329 resolves this and the changes have been published in the [7.9.1 release](https://github.com/okta/okta-signin-widget/releases). Thank you for the report and detailed description!
